config/config.go

```diff
@@ -164,6 +170,14 @@ func (c *HorizonConfig) IsVaultConfigured() bool {
 	return c.AgreementBot.Vault != VaultConfig{}
 }
 
+func (c *HorizonConfig) GetSecretsManagerFilePath() string {
+	path := c.Edge.SecretsManagerFilePath
+	if path == "" {
+		path = "/var/horizon/secrets"
```

Comment:
Change GetSecretsManagerFilePath, I didn't change GetFileSyncServiceAPIListen() since it's default is /var/run/horizon not /var/horizon like HZN_VAR_BASE_DEFAULT